### PR TITLE
Added logo and icon in differents formats

### PR DESCRIPTION
**Pull request by nucliweb**
- Added logo in vector and bitmap format
- Added icon in vector and bitmap format

### CODE REVIEW

**Comment by nwinkler** [comment]
Great - thanks!

**Comment by nucliweb** [comment]
😄

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
-# media
-Bash-it media files
+![Bash-it](Bash-it.png)
```
Comment (rayj0) [inline README.md:1]: Looks great

File: media/Bash-it_icon.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg id="Bash_it" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><style>.st0{fill:#2e3c3c}.st1{fill:#fff}.st2{fill:#79ac59}</style><path class="st0" d="M502 512H10c-5.5 0-10-4.5-10-10V10C0 4.5 4.5 0 10 0h492c5.5 0 10 4.5 10 10v492c0 5.5-4.5 10-10 10z"/><path class="st1" d="M331.9 230.4v-50.8h-49.5V130h-50.8v49.5h-52.1V130h-50.8v49.5H79.2v50.8h49.5v52.1H79.2v50.8h49.5v49.5h50.8v-49.5h52.1v49.5h50.8v-49.5h49.5v-50.8h-49.5v-52.1h49.5zm-100.3 52h-52.1v-52.1h52.1v52.1z"/><path class="st2" d="M382.7 130h50.8v152.4h-50.8zM382.7 333.2h50.8v49.5h-50.8z"/></svg>
```
Comment (rayj0) [inline media/Bash-it_icon.svg:1]: media/Bash-it_400px_white.png

Comment (rayj0) [reply media/Bash-it_icon.svg:1]: Looks great

**Review by rayj0 — APPROVED** [review]

**Pull request merged**